tests/ballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -95,7 +95,25 @@ under the License.
                     <exclude name="testOptionalArgsInNativeFunc" />
                 </methods>
             </class>
+            <class name="org.ballerinalang.test.object.ObjectTest" >
+                <methods>
+                    <!-- fails due to functions without a body -->
+                    <exclude name="testAbstractClientObject" />
+                    <exclude name="testObjectWithAttachedFunction1" />
+                    <exclude name="testObjectWithMissingNativeImpl" />
+
+                    <!-- fails in getBIRModuleBinary -->
+                    <exclude name="testObjectAnyTypeFieldAsConstructorParam" />
+                    <exclude name="testObjectWithAttachedFunction" />
+                    <exclude name="testStructPrint" />
+
+                    <!-- fails because of missing lambda functions support -->
+                    <exclude name="testFunctionReferencesFromObjects" />
+                </methods>
+            </class>
             <class name="org.ballerinalang.test.object.AnonymousObjectTest" />
+            <class name="org.ballerinalang.test.object.ObjectEquivalencyNegativeTest" />
+            <class name="org.ballerinalang.test.object.ObjectEquivalencyNegativeTest" />
```

Comment:
I have mistakenly added a duplicate entry. This is removed in the next PR: #15018